sale_order_partner_no_autofollow/models/sale_order.py

```diff
@@ -0,0 +1,45 @@
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def message_subscribe(self, partner_ids=None, channel_ids=None, subtype_ids=None):
+        if self.env.context.get("sale_partner_disable_autofollow", False):
```

Comment:
Why do you not use check `customer_no_autofollow` config parameter here?
If you are checking config parameter  here, then `create`, `action_confirm` the functions won't make sense.